physcraper/__init__.py

```diff
@@ -1381,20 +1320,20 @@ def get_ncbiid_from_tax_name(self, tax_name):
                                      "correctly: {}! We set it to unidentified".format(tax_name))
                     tax_name = 'unidentified'
                     ncbi_id = 0
-                    # TODO: ADD otudict entries....MK: wrong class, otudict is in ATT
         assert type(ncbi_id) is int
         self.spn_to_ncbiid[tax_name] = ncbi_id
         return ncbi_id
 
     def get_rank_info_from_web(self, taxon_name):
```

Comment:
Because, as we decided to have the web based light version as a second project, I did not yet change it to use an id instead.

Reply:
makes sense.